antora-ui-camel/src/css/blog.css

```
@@ -154,3 +154,10 @@ article.blog p {
     flex-direction: row;
   }
 }
+
+/* for mobile screens */
+@media screen and (max-width: 1023px) {
+  .doc aside {
```

This should be scoped to `.blog` instead.
```suggestion
  .blog aside {
```